apps/aebytecode/src/aebytecode.app.src

```diff
@@ -1,15 +0,0 @@
-{application, aebytecode,
- [{description, "Bytecode definitions for AEthernity VM shared with compiler."},
-  {vsn, "0.1.0"},
-  {registered, []},
-  {applications,
-   [kernel,
-    stdlib,
-    lager
```

Comment:
I see lager removed among applications: intended?

Reply:
Yes, this one is not depending on lager... it's just a very small library